ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -280,7 +280,7 @@ tests:
     cluster_profile: aws
     upgrade: false
 - as: launch-aws
-  commands: sleep 7200 & wait
+  commands: sleep 10800 & wait
```

Comment:
This is something else, not e2e tests.

Reply:
ok i've removed that change.

Reply:
The other one is a launch-aws job too.  More background on ot [here][1].

[1]: https://github.com/openshift/release/pull/2108#issue-229997735